submarine-server/server-api/src/main/java/org/apache/submarine/server/api/spec/EnvironmentSpec.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.submarine.server.api.spec;
+
+/**
+ * Submarine environment spec.
+ */
+/**
+ * @author mani
+ *
+ */
+public class EnvironmentSpec {
+
+  
+  /**
+   * Name of the environment
+   */
+  private String name;
```

Comment:
We have name in both Environment and EnvironmentSpec, should we dedup it? (I suggest to leave it in the Environment only).

Reply:
Ok. Since name is part of EnvironmentSpec, kept in EnvironmentSpec class itself.